user-guide/Getting_started/The_DataMiner_User_Interface/DataMiner_Cube/Cube_UI_components_X/DataMiner_Cube_sidebar.md

```diff
@@ -166,14 +166,18 @@ The following actions are possible with this list:
 
 The apps button displays a list of DataMiner applications.
 
-Depending on the configuration of your DataMiner System, the list can contain up to four sections:
+Depending on the configuration and version of your DataMiner System and on the available user permissions, the list can contain the following sections:
```

Comment:
configuration, version of your DataMiner System, and on the available user permissions...

Reply:
In this case, "configuration and version of your DataMiner System" are placed together because it is "the configuration of your DataMiner System". I'm not sure if "the configuration" on its own really makes much sense. However, if it's the two "ands" you'd like to get rid of, I could rephrase this like this:
```suggestion
Depending on the configuration and version of your DataMiner System, as well as on which user permissions you have been assigned, the list can contain the following sections:
```